checkQC/run_type_recognizer.py

```diff
@@ -113,11 +105,20 @@ def __init__(self, config, runfolder):
         """
         self._config = config
         self._runfolder = runfolder
-        with open(os.path.join(self._runfolder, "RunInfo.xml")) as f:
-            self.run_info = xmltodict.parse(f.read())
+        try:
+            run_info_path = os.path.join(self._runfolder, "RunInfo.xml")
+            if not os.path.exists(run_info_path):
+                log.error("Could not find a RunInfo.xml in {}. Are you sure this is a runfolder?".format(run_info_path))
```

Comment:
Shouldn't this also raise an error?

Reply:
You are right, that might be better. Right now it is raised by the `open` statement below instead, but it might be better to do it explicitly instead.

Reply:
:+1: